### PR TITLE
chore: delete unused *.proto files

### PR DESCRIPTION
**Pull request by busunkim96**
`.proto` files were initially copied when we still used the monolithic gapic-generator and the generated`_pb2.py` files were not  human readable. These files no longer serve a purpose and can be removed.

### CODE REVIEW

**Comment by busunkim96** [comment]
The regex replacement I added was not idempotent and now owlbot is continuing to make the same replacement 😅 . I'll adjust and re-open this PR.

File: tests/unit/gapic/securitycenter_v1beta1/test_security_center.py [hunk 1]
```diff
@@ -3622,7 +3622,7 @@ def test_set_finding_state_flattened():
         _, args, _ = call.mock_calls[0]
         assert args[0].name == "name_value"
         assert args[0].state == finding.Finding.State.ACTIVE
-        # assert args[0].start_time == timestamp_pb2.Timestamp(seconds=751)
+        assert args[0].start_time == timestamp_pb2.Timestamp(seconds=751)
```
Comment (parthea) [inline tests/unit/gapic/securitycenter_v1beta1/test_security_center.py:3625]: Did you mean to uncomment this line?

Comment (busunkim96) [reply tests/unit/gapic/securitycenter_v1beta1/test_security_center.py]: Nope! Fixed, PTAL.

**Comment by busunkim96** [comment]
I think this should be good now (I accidentally deleted an empty line and lint failed 🤦‍♀️ ).

**Review by parthea — APPROVED** [review]

**Pull request merged**